### PR TITLE
Name change in different files

### PR DESCRIPTION
**Pull request by sarthak-sopho**
With respect to our discussions in #7

### CODE REVIEW

**Comment by uniqueg** [comment]
Thanks @sarthak-sopho. Unfortunately CI pipeline fails for Python 3.7. Seems like a dependency is missing

File: requirements.txt [hunk 1]
```diff
@@ -1,35 +1,9 @@
...
-six==1.14.0
-typed-ast==1.4.1
-wcwidth==0.1.8
-Werkzeug==1.0.0
-wrapt==1.11.2
-zipp==3.1.0
+Flask==1.1.1
+pylint==2.4.4
+pytest==5.4.1
+PyYAML==3.12
```
Comment (uniqueg) [inline requirements.txt]: This regression (4.2b1 to 3.12) is probably the reason for the CI failing

File: requirements.txt [hunk 2]
```diff
@@ -1,35 +1,9 @@
...
+requests==2.18.4
```
Comment (uniqueg) [inline requirements.txt]: Isn't this built-in? Don't think it's necessary to list explicitly

**Comment by sarthak-sopho** [comment]
Made changes :)

**Pull request merged**